stylus/ui-app/notifications.styl

```diff
@@ -0,0 +1,54 @@
+/*!
```

Comment:
Minus things in this file: externalize colors, `em()` function, `embedurl`… Like in those above :)